### PR TITLE
(Archiving) Modify according to documentation

### PR DESCRIPTION
**Pull request by islean**
## Description

With the catalogue and data mover servers being up and running, we need to modify some parts of the flow, now that we can take part of the documentation. This is a pre-PR needed to test some functionality when the CLI functionality is added as well.

### Changed

- Get job status endpoint, request and response.
- Aliases for camel case responses.

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b archiving-correct-requests-responses -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/meta/archive/ddn_dataflow.py [hunk 1]
```diff
@@ -172,60 +177,28 @@ class TransferJob(BaseModel):
...
-    subjob_id: Optional[int] = None
-    related_jobs: Optional[bool] = None
-    main_subjob: Optional[bool] = None
-    debug: Optional[bool] = None
+    id: int
 
-    def post_request(self, url: str, headers: dict) -> GetJobStatusResponse:
-        """Sends a request to the given url with the given headers, and its own content as
-        payload. Returns the job ID of the launched transfer task.
+    def get_status(self, url: str, headers: dict) -> GetJobStatusResponse:
```
Comment (henrikstranneheim) [inline cg/meta/archive/ddn_dataflow.py:195]:
```suggestion
    def get_job_status(self, url: str, headers: dict) -> GetJobStatusResponse:
```

File: cg/meta/archive/ddn_dataflow.py [hunk 2]
```diff
@@ -172,60 +177,28 @@ class TransferJob(BaseModel):
...
+        """Sends a get request to the given url with the given headers.
```
Comment (henrikstranneheim) [inline cg/meta/archive/ddn_dataflow.py:196]:
```suggestion
        """Sends a get request to the given URL with the given headers.
```

File: cg/meta/archive/ddn_dataflow.py [hunk 3]
```diff
@@ -172,60 +177,28 @@ class TransferJob(BaseModel):
...
+        Returns the parsed status response of the task specified in the URL.
         Raises:
-             HTTPError if the response code is not ok.
-        """
+             HTTPError if the response code is not ok."""
```
Comment (henrikstranneheim) [inline cg/meta/archive/ddn_dataflow.py:199]:
```suggestion
             HTTPError if the response code is not successful."""
```

File: tests/meta/archive/conftest.py [hunk 4]
```diff
@@ -45,13 +45,13 @@ def ddn_dataflow_config(
 
 @pytest.fixture
 def ok_ddn_response(ok_response: Response):
-    ok_response._content = b'{"job_id": "123"}'
+    ok_response._content = b'{"jobId": "123"}'
     return ok_response
 
 
 @pytest.fixture
 def ok_ddn_job_status_response(ok_response: Response):
```
Comment (henrikstranneheim) [inline tests/meta/archive/conftest.py:53]:
```suggestion
def ok_miria_job_status_response(ok_response: Response):
```

**Review by henrikstranneheim — APPROVED** [review]
Nice 👍

**Review by Vince-janv — APPROVED** [review]
Great 👍

**Pull request merged**